### PR TITLE
Small improvements

### PR DESCRIPTION
**Pull request by v-chmykov**
Hi guys!

It's a great idea to make this project. I wanna wish you good luck and lots of exciting projects! 
I found one weird behaviour in your contact form - error message about incorrect email is always visible. So in this PR, I made this error message visible when input email is invalid.
Besides, I made some small improvements:
- Created a list of contents in stylesheets for better development experience.
- Moved your js from _index.html_ to a separate file.
- Formatted _index.html_ for better readability (in my opinion). We can discuss this if it does not make sense for you.

Looking forward to hear any feedback from you.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -280,136 +317,128 @@ <h3 class="name">Rajesh Ragul</h3>
...
           <div class="social"><a href="mailto:ruthvikchanda1999@gmail.com"><i class="fa fa-envelope"></i></a><a href="https://m.facebook.com/ruthvik.chanda"
               target="_blank"><i class="fa fa-facebook-official"></i></a><a href="https://www.linkedin.com/in/ruthvik-chanda-08304615b"
               target="_blank"><i class="fa fa-linkedin"></i></a></div>
         </div>
-
       </div>
     </div>
   </div>
 
-  <!--
```
Comment (vchrombie) [inline index.html:355]: start

File: index.html [hunk 2]
```diff
@@ -280,136 +317,128 @@ <h3 class="name">Rajesh Ragul</h3>
...
-                        <h5 class="name">Emily Clark</h5>
-                        <p class="title">Owner of Creative Ltd.</p>
-                    </div>
-                </div>
-            </div>
-        </div>
-    </div>
-
-    <div class="brands" id="brands"><a href="#"> <img src="assets/img/instacart.png"><img src="assets/img/kickstarter.png"><img src="assets/img/lyft.png"><img src="assets/img/shopify.png"><img src="assets/img/pinterest.png"><img src="assets/img/twitter.png"></a></div>
--->
```
Comment (vchrombie) [inline index.html]: end

**Comment by vchrombie** [comment]
Hey, @v-chmykov 
Thanks for the PR, it looks great but I need that Testimonials and Brands sections for later use. 
It would be good if you can add only both those and the rest of the formatting is cool.

Looking forward to your PR. :)

**Comment by v-chmykov** [comment]
Done - restored it :)
Sorry,  I didn't know that you are going to use it.

**Pull request merged**